charts/kube-prometheus-stack/templates/prometheus-operator/clusterrole.yaml

```diff
@@ -103,7 +103,10 @@ rules:
   - endpointslices
   verbs:
   - get
+  - create
```

Comment:
Ideally we can use the `kubeletEndpointsEnabled` and `kubeletEndpointSliceEnabled` to toggle granting these permissions. will let reviewer decide. i can add this logic if needed

Reply:
The toggle would cover only write permissions, right?

Reply:
No, I think all of the endpointslice permissions. unelss there is a reason not to (I dont see one)